src/components/swap/GasBreakdownTooltip.tsx

```diff
@@ -69,11 +70,13 @@ export function GasBreakdownTooltip({ trade, hideUniswapXDescription }: GasBreak
   )
 }
 
-function NetworkFeesDescription({ native }: { native: Currency }) {
+function NetworkCostDescription({ native }: { native: Currency }) {
+  const chainName = chainIdToBackendName(native.chainId)
+
   return (
     <ThemedText.LabelMicro>
       <Trans>
-        The fee paid to the Ethereum network to process your transaction. This must be paid in {native.symbol}.
+        Network cost is paid in {native.symbol} on the {chainName} network in order to transact.
```

Comment:
I actually don't know how this works but just curious - i imagine `<Trans>` relies on the inner string contents to do the translation, but here we have multiple slots that are filled in eg `{native.symbol}`, does Trans still work?

Reply:
not sure who is best suited to answer maybe @zzmp

Reply:
so lingui is smart enough to extrapolate variables inside jsx correctly, see the example on their home page :D 
https://lingui.dev/introduction